### ISSUE TITLE
Change userName?

### ISSUE THREAD

**Issue by jonahbron** [nix-community/nix-on-droid#320]
I found the option for setting the username of the account used.

https://github.com/nix-community/nix-on-droid/blob/039379abeee67144d4094d80bbdaf183fb2eabe5/modules/user.nix#L57

But it is set to readonly.  Is there an important reason that's not configurable, or is there some other avenue to set it?

**Comment by cfitz25** [comment]
im curious too as i think this causes a username collision with a username i have set in home-manager

**Comment by boozedog** [comment]
I would love to change username for both home-manager and making outbound ssh easier

**Comment by osbm** [comment]
Yeah setting the username would help a lot of people. What are the maintainers thought on this?

Should one of us just open a PR?

**Comment by t184256** [comment]
I don't remember/know why it's readonly; feel free to try and see what breaks.

Outbound SSH is kind of of a weak argument, because there's `programs.ssh.matchBlocks.<name>.user`. I don't know what "set username in home-manager" means, so it'd be nice if somebody explained it to me.

**Comment by osbm** [comment]
i just set readOnly to false in my fork and set the userName variable with lib.mkForce in my config and everything works as far as i can tell.
